app/tests/unit/pages/project-revision/[projectRevision]/form/annual-reports.test.tsx

```diff
@@ -0,0 +1,145 @@
+import "@testing-library/jest-dom";
+import { screen, within } from "@testing-library/react";
+import { mocked } from "jest-mock";
+import { useRouter } from "next/router";
+import { getProjectRevisionPageRoute } from "pageRoutes";
+import { ProjectAnnualReportsPage } from "pages/cif/project-revision/[projectRevision]/form/annual-reports";
+import PageTestingHelper from "tests/helpers/pageTestingHelper";
+import compiledAnnualReportsFormQuery, {
+  annualReportsFormQuery,
+} from "__generated__/annualReportsFormQuery.graphql";
+
+jest.mock("next/dist/client/router");
```

Comment:
If https://github.com/bcgov/cas-cif/pull/605 goes in before this PR, there will be router-mocking helpers available